src/components/Modal/__tests__/Modal.test.jsx

```diff
@@ -368,30 +349,6 @@ describe.skip('functionality', () => {
     assertFocus(el, true);
   });
 
-  it('traps focus', async () => {
```

Comment:
HTML dialog handles focus trap manually, so there is no need to test it.  What more, it does not work in testing environment as it is not implemented.